custom/icds_reports/models/aggregate.py

```diff
@@ -1550,3 +1551,21 @@ class Meta(object):
 
     _agg_helper_cls = AggAdolescentGirlsRegistrationAggregate
     _agg_atomic = False
+
+
+class AggregateVHNDForms(models.Model, AggregateMixin):
+    state_id = models.TextField(null=True)
+    vhsnd_date_past_month = models.DateField(null=True)
+    anm_mpw_present = models.NullBooleanField(null=True)
+    asha_present = models.NullBooleanField(null=True)
+    child_immu = models.NullBooleanField(null=True)
+    anc_today = models.NullBooleanField(null=True)
+    awc_id = models.TextField(null=True)
```

Comment:
would be good to mark Awc_id  as primary key else django will add primary key column by itself

Reply:
done